### ISSUE TITLE
Change the term `scratch cluster` to `draft cluster` or add some explanation/definition to it?

### ISSUE THREAD

**Issue by winkee01** [kubernetes/website#30448]
Issue with k8s.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/

The term `scratch cluster` that appears in the description might cause a bit of confusion (to me at least). Would it be better to rephrase it to something like `exercise cluster` or `draft cluster`?

![CleanShot 2021-11-12 at 10 21 41@2x](https://user-images.githubusercontent.com/49930785/141397899-adddbe3a-a6ef-401a-8749-4f8220f52fd2.png)

**Comment by AvineshTripathi** [comment]
I too think we can have a line defining scratch work and then maybe we can talk about the cluster to make thing uniform through out

**Comment by shannonxtreme** [comment]
Scratch work usually means quick scribble work. Like, say, you have a maths problem and you're just thinking of ways to solve it and you write stuff on a whiteboard and rub it out if it isnt right etc. That's scratch work. I do not think this is a good term to use in docs that are localized downstream without an explanation (@tengqm idk if you agree). 

We should either explain what "scratch" means in this context very clearly, or we should replace scratch with something like "rough work" or "a cluster for brainstorming and problem-solving work", or similar.

**Comment by tengqm** [comment]
I don't have a strong opinion on this. The term "scratch" is not an uncommon word. Given the context, the meaning is pretty apparent. However, it doesn't hurt if we can make it clearer.  :)

**Comment by sftim** [comment]
This looks more like a support query at the moment @winkee01 
Would you be willing to edit the issue description and rephrase it as a request to clarify or reword the (jargon) term “scratch”, in that page?

/language en

**Comment by sftim** [comment]
We could also add an HTML comment after “scratch” to give context for localizers.

**Comment by sftim** [comment]
/help

Folks are welcome to reply to this comment if anything isn't clear.

**Comment by reylejano** [comment]
@winkee01 , in the original paragraph:

> Suppose you have two clusters, one for development work and one for scratch work. In the development cluster, your frontend developers work in a namespace called frontend, and your storage developers work in a namespace called storage. In your scratch cluster, developers work in the default namespace, or they create auxiliary namespaces as they see fit. Access to the development cluster requires authentication by certificate. Access to the scratch cluster requires authentication by username and password.

Is the phrase "scratch work" clear?
If it is not, then is "test work" or "temporary work" better? Or are "test work" or "temporary work" too close to "development work"?

**Comment by winkee01** [comment]
To me, changing from `scratch xxx` to `test xxx` has a clearer meaning.

**Comment by shannonxtreme** [comment]
test cluster feels nearer to me as well. Especially paired with temporary work. An intro that defines it like "Suppose you have two clusters, a dev cluster for development work and a test cluster for temporary work such as brainstorming and problem solving"

**Comment by divya-mohan0209** [comment]
For anyone picking up this issue
- Please read the entire discussion above, particularly [this comment](https://github.com/kubernetes/website/issues/30448#issuecomment-1164970348) by @reylejano 
- Kindly ask questions, if any, on the issue.

**Comment by dipesh-rawat** [comment]
Changes to address this issue have been merged as part of #39656

**Comment by dipesh-rawat** [comment]
Done

/close
